commons/src/main/java/org/calrissian/accumulorecipes/commons/hadoop/BaseQfdInputFormat.java

```diff
@@ -72,6 +74,15 @@ protected static void configureScanner(Job job, Node query, GlobalIndexVisitor g
         setting.addOption(BooleanLogicIterator.QUERY_OPTION, originalJexl);
         setting.addOption(BooleanLogicIterator.FIELD_INDEX_QUERY, jexl);
 
+        final String[] selectFields = job.getConfiguration().getStrings("selectFields");
+
+        setting.addOption("auths", getScanAuthorizations(job).serialize());
+        if (selectFields != null && selectFields.length > 0) {
+            IteratorSetting iteratorSetting = new IteratorSetting(16, SelectFieldsExtractor.class);
+            SelectFieldsExtractor.setSelectFields(iteratorSetting, new HashSet<String>(Arrays.asList(selectFields)));
+        }
```

Comment:
You can use guavas Sets.newHashSet(...) here instead.  Only a small thing, but avoids needing to create an ArrayList to simply make a HashSet.